### DISCUSSION TITLE
[Art competition] SQL Injection

### DISCUSSION THREAD

**Discussion by JOYB28** [prosyslab-classroom/cs348-information-security#412] [Art competition]
<image width=700 src="https://github.com/prosyslab-classroom/cs348-information-security/assets/29791719/4b8c035b-005e-433e-acf2-ef277f684291" />

## Short Description
For me, **SQL Injection** means **A jigsaw puzzle piece with bomb on it on the puzzle**.

## Long Description (up to 3 sentences)
SQL injection is like putting a piece of puzzles with a bomb in it. Attackers attack security vulnerabilities by injecting bomb-like SQL statements inside requests. To prevent SQL Injection, we need to check that the request (puzzle) does not have hazards (bombs).

**Comment by KihongHeo** [comment]
Nice try. Does the bomb explode once the puzzle is completed?
